books/free-programming-books-en.md

```diff
@@ -1,10 +1,9 @@
 ### Index
-
 * [All](#all)
 
-
 ### All
-
 * [English, By Programming Language](free-programming-books-langs.md)
-* [English, By Subject](free-programming-books-subjects.md)
-  (The list of books in English is here for historical reasons.)
+* [English, By Subject](free-programming-books-subjects.md)(The list of books in English is here for historical reasons.)  
+
+### Python
+* [Python programming language](https://en.wikibooks.org/wiki/Subject%3APython_programming_language) - wikibooks
```

Comment:
Please move the resource to the English by programming language file. Also check the spacing and ordering of the entries. Read [Contributing.md](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md) for more information